action/junit/report.junit

```diff
@@ -0,0 +1,1820 @@
+<testsuites tests="904" failures="3">
```

Comment:
Seems like a good opportunity to trim this down, can we just make a dummy `<testsuite>` with one pass, one skip, and one failure? This file is huge and has 901 passes but doesn't have any skips so it's not comprehensive.

Reply:
Was planning that after getting the CI up